scripts/load_yaml.py

```diff
@@ -0,0 +1,90 @@
+#!/bin/python3
+
+import sys
+import yaml
+
+def print_usage():
+    print("usage: load_yaml.py [filename] [prefix(optional)]")
+    print("       After execution, source the output to add the variables")
+    print("       to the environment.")
+    exit(-1)
```

Comment:
Hier ist noch ein `exit(-1)` in der Usage-Funktion.

Reply:
Hier würde ich ein rebase machen, wenn die richtige Version aus #54 gemergt ist.